packages/contracts/contracts/vesting/Vesting.sol

```diff
@@ -39,6 +41,12 @@ contract Vesting is OwnableUpgradeable, ReentrancyGuardUpgradeable, IVesting {
         uint256 totalAmount,
         IAtomicSwapBase.Release[] memory releases
     ) external payable {
+        // Ensure the uniqueness of 'beneficiary-orderId', to avoid an override call attack.
+        if (vestingSchedules[beneficiary][orderId].from != address(0)) {
+            revert IAtomicSwapBase.DuplicateReleaseSchedule();
+        }
```

Comment:
I think this issue will still exist on inchain atomic side, we can store uuid in contract to avoid duplication issue on both contracts.
See PR #27 , I have added the issue and solution.

Reply:
orderId is just unique always. 
I guess there is no need to make addtional complexity.